modulemd/include/private/modulemd-util.h

```diff
@@ -352,6 +358,23 @@ gboolean
 modulemd_fnmatch (const gchar *pattern, const gchar *string);
 
 
+/**
```

Comment:
This `modulemd_rpm_match()` definition seems to have been added to this header file by a different commit that the one adding the implementation. That might cause problems with bisection...

Reply:
Oops. I'll fix that.